distribution/sender/routes/db-sync-route.xml

```diff
@@ -4,100 +4,61 @@
         <from uri="direct:out-bound-db-sync" />
 
         <log message="Start db-sync-route: ${body}" loggingLevel="DEBUG" />
-
+        
```

Comment:
Why do you need this property? Try as much to avoid exchange properties, they can easily be duplicated in routes which can cause obscure bugs.

Reply:
I have removed the "isValid" exchange property during a refactor. Thanks.